feat(cdp): add docs for the posthog replicator destination (#10926)
PostHog/posthog.com: contents/docs/cdp/destinations/posthog.md

---
title: Send PostHog event data to another PostHog instance
templateId:
    - template-posthog-replicator
---

import Requirements from "../_snippets/requirements.mdx"
import FeedbackQuestions from "../_snippets/feedback-questions.mdx"
import PostHogMaintained from "../_snippets/posthog-maintained.mdx"

<Requirements />

You'll also need access to the destination PostHog account.

## Installation

> 1. In PostHog, click the "[Data pipeline](https://us.posthog.com/pipeline/overview)" tab in the left sidebar.

Reviewer: ```suggestion
1. In PostHog, click the **[Data pipelines](https://us.posthog.com/pipeline/overview)** tab in the left sidebar.
```

Bolds for UI elements

> 2. Click the 'Destinations' tab.

Reviewer: ```suggestion
2. Click the **Destinations** tab.
```

> 3. Search for 'PostHog' and select the destination.

Reviewer: ```suggestion
3. Search for **PostHog** and select the destination.
```

4. Add the Host and API Key of the destination at the configuration step.

> 5. Press 'Create & Enable' and watch your 'Events' list get populated in the destination PostHog instance!

Reviewer: ```suggestion
5. Press **Create & Enable** and watch your 'Events' list get populated in the destination PostHog instance!
```
